.gitignore

```diff
@@ -32,3 +32,5 @@ secrets/
 
 # code-workspaces
 *.code-workspace
+
+.gitignore
```

Comment:
Ignoring the `.gitignore` cannot be right...

Reply:
done